src/main/java/com/epam/grid/engine/provider/utils/slurm/healthcheck/ShowConfigCommandParser.java

```diff
@@ -129,19 +128,11 @@ private static LocalDateTime getStartTime(final List<String> stdOut) {
         return stdOut.stream().filter(out -> out.contains(BOOT_TIME)).findAny()
                 .map(bootTimeStr -> bootTimeStr.split(EQUAL_SIGN))
                 .map(splitStr -> splitStr.length == 2 ? splitStr[1].trim() : "")
-                .map(start -> tryParseStringToLocalDateTime(start, DateTimeFormatter.ofPattern(SLURM_DATE_FORMAT)))
+                .map(start -> DateUtils.tryParseStringToLocalDateTime(start,
```

Comment:
Let's add the overloaded version of `DateUtils.tryParseStringToLocalDateTime()`, that accepts 2 arguments:
1. Date to parse
2. Formatter pattern

Call `DateTimeFormatter.ofPattern` inside and pass it to existing `DateUtils.tryParseStringToLocalDateTime()` implementation

Reply:
done